swap/api_test.go

```diff
@@ -112,38 +113,42 @@ func TestBalances(t *testing.T) {
 	defer clean()
 
 	// test balances are empty
-	testBalances(t, swap, map[enode.ID]int64{})
+	testBalances(t, swap, map[enode.ID]*int256.Int256{})
 
 	// add peer
 	testPeer := addPeer(t, swap)
 	testPeerID := testPeer.ID()
 
 	// test balances with one peer
-	setBalance(t, testPeer, 808)
-	testBalances(t, swap, map[enode.ID]int64{testPeerID: 808})
+	balancePeer := int256.Int256From(808)
+	setBalance(t, testPeer, int256.Int256From(808))
+	testBalances(t, swap, map[enode.ID]*int256.Int256{testPeerID: balancePeer})
 
 	// add second peer
 	testPeer2 := addPeer(t, swap)
 	testPeer2ID := testPeer2.ID()
 
 	// test balances with second peer
-	setBalance(t, testPeer2, 123)
-	testBalances(t, swap, map[enode.ID]int64{testPeerID: 808, testPeer2ID: 123})
+	balancePeer2 := int256.Int256From(123)
+	setBalance(t, testPeer2, int256.Int256From(123))
+	testBalances(t, swap, map[enode.ID]*int256.Int256{testPeerID: balancePeer, testPeer2ID: balancePeer2})
 
 	// test balances after balance change for peer
-	setBalance(t, testPeer, 303)
-	testBalances(t, swap, map[enode.ID]int64{testPeerID: 303, testPeer2ID: 123})
+	balancePeer = int256.Int256From(303)
```

Comment:
my favorite :)

Reply:
i am somewhat partial to `909` as well :stuck_out_tongue: